gravitee-apim-console-webui/src/management/api/creation-v4/steps/step-2-entrypoints/step-2-entrypoints-2-config.harness.ts

```diff
@@ -99,4 +105,10 @@ export class Step2Entrypoints2ConfigHarness extends ComponentHarness {
 
     await this.clickValidate();
   }
+
+  async selectQos(entrypointId: string, qos: Qos) {
+    // const gioFormQos = await this.getQosSelect(entrypointId)();
```

Comment:
Remove comment

Reply:
🤦🏻 forgot to cleanup this during my tests